WordPress/src/main/java/org/wordpress/android/ui/accounts/AbstractFragment.java

```diff
@@ -63,12 +63,12 @@ protected boolean onDoneEvent(int actionId, KeyEvent event) {
         return false;
     }
 
-    protected boolean didPressNextKey(int actionId, KeyEvent event) {
+    public static boolean didPressNextKey(int actionId, KeyEvent event) {
```

Comment:
I wonder there is a better place for these 2 methods. If I was looking for this functionality. I wouldn't think to look to `AbstractFragment`. I don't really have a suggestion though, just thinking out loud, I guess.

Edit: I just noticed that these 2 methods are the only `static` ones we have in `AbstractFragment`, which I guess can be a good reason to think about moving these elsewhere.